results/go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/google/go-cmp v0.4.0
 	github.com/mattn/go-sqlite3 v2.0.3+incompatible
 	github.com/tektoncd/pipeline v0.12.0
+	github.com/yuege01/experimental v0.7.0 // indirect
```

Comment:
This shouldn't be needed, since everything should be relative to the current module. We should remove this.

Reply:
done